SpywareFilter/sections/tracking_servers.txt

```diff
@@ -76,6 +76,10 @@
 ||us.tags.newscgp.com^
 ||smazaz.icu^
 ||mpraven.org^
+||beacon.cdn.qq.com^
+||beacon.qq.com^
+||aegis.qq.com^
+||rumt-zh.com^
```

Comment:
Not sure about rumt-zh.
```suggestion
```

Reply:
It is used by aegis-web-sdk to upload data.

> https://www.npmjs.com/package/aegis-web-sdk